yarn-project/foundation/src/transport/browser/worker_listener.ts

```diff
@@ -10,7 +10,7 @@ declare interface DedicatedWorkerGlobalScope {
   /**
    * Handler for incoming messages from other threads or workers.
    */
-  onmessage: (...args: any) => any;
```

Comment:
Why this change? The function type is somewhat more accurate

Reply:
That change was made in the [wasm's version](https://github.com/AztecProtocol/aztec-packages/blob/83a0f633dc9d816f022d3ace009293107b3f261c/yarn-project/foundation/src/wasm/transport/browser/shared_worker_listener.ts) and without it, it doesn't compile. I didn't dig into it because [the function](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/foundation/src/wasm/worker/browser/start_web_module.ts#L8) which uses it is not currently used anywhere.

After looking at it it looks incorrect 🤔